src/components/periodic-report/periodic-report.repository.ts

```diff
@@ -111,17 +111,72 @@ export class PeriodicReportRepository extends DtoRepository(IPeriodicReport) {
       );
   }
 
+  async currentProjectReport(
+    projectId: string,
+    reportType: ReportType,
+    startDate: DateTime
```

Comment:
```suggestion
    startDate: CalendarDate
```

Use CalendarDates instead of DateTimes. This is the property type being matched on report (start date) and you won't need to cast to a date in cypher this way (`date($date)`).